Register rishukumar.is-a.dev

rishukumark19 opened the pull request:
# Requirements
- [x] I have **read** and **agreed** to the [Terms of Service](https://is-a.dev/terms).
- [x] My file is following the [domain structure](https://docs.is-a.dev/domain-structure/).
- [x] My website is **reachable** and **completed**.
- [x] My website is **software development** related.
- [x] My website is not for commercial use.
- [x] I have provided sufficient contact information in the `owner` key.
- [x] I have provided a preview of my website below.

# Website Preview
I am registering my developer portfolio subdomain at rishukumar.is-a.dev which will point to my GitHub Pages site at rishukumark19.github.io. This will be used to showcase my web development projects and portfolio.

rishukumark19 committed:
Register rishukumar.is-a.dev

wdhdev reviewed (dismissed):
Please provide a website preview.

wdhdev committed:
Refactor JSON structure in rishukumar.json

Formatted JSON for better readability and corrected 'record' to 'records'.

rishukumark19 committed:
Fix username to match GitHub account (rishukumark19)

wdhdev commented:
[rishukumar.github.io](https://rishukumar.github.io/) is not accessible

rishukumark19 committed:
Fix CNAME: Update to correct GitHub Pages URL

Changed CNAME from 'rishukumar.github.io' to 'rishukumark19.github.io' to match the actual GitHub Pages repository URL. The previous URL was not accessible.

iostpa reviewed (changes requested):
Fix the profile icon not loading and the projects section and I think you should be good to go.

rishukumark19 commented:
> Fix the profile icon not loading and the projects section and I think you should be good to go.
Thanks for your feedback! I’ve made the changes—please check it out and let me know if you like it :)

iostpa reviewed (approved):
Seems good to me now. Also the website looks nice.
